README.Rmd

```diff
@@ -19,7 +19,7 @@ ver <- paste0("https://img.shields.io/badge/Version-", ver,
 ```
 
 
-# `SomaScan.db` <img src="inst/figures/logo.png" align="right" height="138" alt="" />
+# `SomaScan.db`
```

Comment:
If you remove this from here (even though BioC doesn't like it), won't the logo will be missing from the GH README page?

Reply:
Yes, it does end up being removed from the GH README, but if I don't remove it then I get a broken image icon next to the package title in the pkgdown site. It seems like the only way to satisfy both situations (GH README _and_ pkgdown landing page) is to remove the image, unfortunately.

Reply:
Here is a screenshot of the pkgdown site, for reference: ![image](https://github.com/SomaLogic/SomaScan.db/assets/29712682/de25e7c2-c227-4aa2-b5d0-f659d5789a54)